src/client/components/FriendInfo/index.js

```diff
@@ -0,0 +1,21 @@
+'use strict';
+
+import React from 'react';
+
+class FriendInfo extends React.Component {
```

Comment:
try to use [functional component](https://javascriptplayground.com/blog/2017/03/functional-stateless-components-react/), just for practicing